Game/Source/GDKTestGyms/Private/UserExperienceComponent.cpp

```diff
@@ -76,6 +82,21 @@ void UUserExperienceComponent::OnClientOwnershipGained()
 	GetWorld()->GetTimerManager().SetTimer(Timer, Delegate, 1.0f, true);
 }
 
+float UUserExperienceComponent::CalculateAverageVL() const
+{
+	const float NCDSquared = GetOwner()->NetCullDistanceSquared;
+
+	float Avg = 0.0f;
+	for (int i = 0; i < UpdateRate.Num(); i++)
+	{
+		const UpdateInfo& UR = UpdateRate[i];
+		check(UR.DistanceSq < NCDSquared);
+		Avg += (UpdateRate[i].DeltaTime * (1.f - UR.DistanceSq / NCDSquared));
```

Comment:
What if we normalize the (1.f - UR.DistanceSq / NCDSquared) values to ensure we keep the units the same?

Reply:
Doesn't the check(UR.DistanceSq < NCDSquared); ensure that that's the case?
Or are you suggesting removing the check in favour of normalizing it?